src/IssueViz.Security/IssueViz.Security.csproj

```diff
@@ -36,6 +45,7 @@
   <ItemGroup Label="Package references">
     <PackageReference Include="Microsoft.VisualStudio.ComponentModelHost" Version="14.0.25424" />
     <PackageReference Include="Microsoft.VisualStudio.Language.Intellisense" Version="14.0.23205" />
+    <PackageReference Include="Microsoft.VisualStudio.Shell.14.0" Version="14.3.25407" />
```

Comment:
I think this should be in the props file too, rather than splitting the logic between the two files.

Reply:
It's not exactly the same. The props file is when you know you need a VS specific version of shell, and it's an addition to your existing references. Here the shell is the basic shell for the entire project. If we remove the WpfTableControlProvider, we only need to remove the RequiresShell flag, and everything else would be the same.
Also, there are additional shell references, like shell interop and shell immutable, which we do/will specify directly. So to me it makes sense to have the original shell right next to them. WDYT?